### ISSUE TITLE
Scraping error: login.menu.key: history/historyAnalysis

### ISSUE THREAD

**Issue by chilek** [jwilk/mbank-cli#68]
Script generates the error as follows:

```
Scraping error: login.menu.key: history/historyAnalysis at /opt/chilan/mbank-cli line 861.
	main::__ANON__(HASH(0x55b0ac627608)) called at /opt/chilan/mbank-cli line 871
	main::__ANON__(HASH(0x55b0ac627278)) called at /opt/chilan/mbank-cli line 891
	main::_extract_login_menu(HTML::TreeBuilder=HASH(0x55b0ac3b65b0)) called at /opt/chilan/mbank-cli line 1019
	main::do_login("config", HASH(0x55b0ac214f28)) called at /opt/chilan/mbank-cli line 2528
Please file a bug at <https://github.com/jwilk/mbank-cli/issues>.
```

I use current master from github.

**Comment by jwilk** [comment]
Thanks for the bug report.
Fixed in 78b64c05052c26709b39d5e3a2d6f882ec0048ba.
I'll make a new release soon.

**Comment by chilek** [comment]
Thanks a lot - it resolved my problem!

**Comment by jwilk** [comment]
Fixed in 1.7.1.
